parlai/core/torch_generator_agent.py

```diff
@@ -1002,6 +1002,9 @@ class BeamSearch(TreeSearch):
 
     def select_paths(self, logprobs, prior_scores):
         """Select the next vocabulary item in these beams."""
+        if prior_scores.numel() == 1:
```

Comment:
Add the comment back explaining the choice

Reply:
ok